### ISSUE TITLE
TmsImage proj argument seems to be ignored

### ISSUE THREAD

**Issue by duckontheweb** [DigitalGlobe/gbdxtools#600]
## GBDXTools New Issue

Please provide some background information:

### GBDX Username

jon.duckworth@digitalglobe.com

### GBDX Account Tier

* [ ] Community (Free)
* [ ] Notebooks Only
* [ ] GBDX Starter
* [ ] GBDX Enterprise
* [x] [other, please clarify]

Internal DG user

### GBDXTools Version

Run `pip show gbdxtools` in a terminal or `!pip show gbdxtools` in a Jupyter notebook cell

Name: gbdxtools
Version: 0.16.1
Summary: API wrapper and imagery access for the GBDX Platform
Home-page: https://github.com/DigitalGlobe/gbdxtools
Author: DigitalGlobe
Author-email: UNKNOWN
License: MIT
Location: /anaconda/envs/gbdx_py3/lib/python3.6/site-packages
Requires: toolz, pycurl, configparser, future, numpy, cloudpickle, mock, boto3, affine, pyproj, ephem, requests-futures, requests, shapely, pytest-runner, vcrpy, pytest, six, gbdx-auth, scikit-image, mercantile, cachetools, bumpversion, dask
Required-by:

### Python Version

* [ ] 2
* [x] 3

### Environment

* [x] GBDX Notebooks
* [x] Local

[ If local, please provide more information about your local Python environment. For example, which Python executable - system or Anaconda? ]

Happens in conda environment and virtualenv environment.

### Description

Using the `proj` argument when instantiating a `TmsImage` instance seems to have no effect. If I use `proj='EPSG:4326'` and save the image using the `geotiff` method it saves an image in the native EPSG:3857.

### Expected behavior:

Image saved as EPSG:4326.

### Actual behavior:

Image saved as EPSG:3857

### Example

```python
from gbdxtools import TmsImage
from os import environ
import rasterio

bbox = [-104.966, 39.711, -104.958, 39.719]
zoom_level = 16

tms = TmsImage(
    zoom=zoom_level, 
    bbox=bbox, 
    access_token=environ['MAPBOX_API_KEY'], 
    proj='EPSG:4326'
    # proj=4326 has same effect
)
tms.geotiff(path='/tmp/test.tif')

with rasterio.open('/tmp/test.tif') as src:
    print(src.crs)

# Prints '+init=epsg:3857'
```

**Comment by chelm** [comment]
Looked at this a bit and we're not currently sure if TMSImage ever actually support the proj param. Its proving to be a bit tricky to support it too. Will keep looking for a solution.

**Comment by chelm** [comment]
In the meantime i'd handle this via `gdal_warp` after its been converted to a tif, if thats helpful.

**Comment by duckontheweb** [comment]
Okay, no problem. We're currently using `gdal_warp` in our pipeline, so no rush on fixing this. It might be good to remove the `proj` argument from the docs just so no one runs into this until it is fixed.

**Comment by chelm** [comment]
@duckontheweb yeah good call. Will do that.

**Comment by drwelby** [comment]
With dask-based warping not working correctly, there's no available solution internal to GBDXtools.
